lighthouse-cli/test/smokehouse/seo/expectations.js

```diff
@@ -42,6 +45,9 @@ module.exports = [
       'meta-description': {
         score: false,
       },
+      'http-status-code': {
+        score: false,
```

Comment:
looks like `displayValue` can also be asserted on this failure?

Reply:
good point, added!